### PR TITLE
Fix typos and update variable usage in delivery messages

### PR DESCRIPTION
**Pull request by karlnyr**
## Description

Just some minor details on the delivery messages that fixes the flow of the language

### Fixed

- Some grammar and sentence structure

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

**Review by seallard — APPROVED** [review]
👍

File: cg/services/delivery_message/messages/scout_message.py [hunk 1]
```diff
@@ -12,7 +12,7 @@ def get_cases_message(cases: list[Case]) -> str:
     scout_links: list[str] = [get_scout_link(case) for case in cases]
     scout_links_row_separated: str = "\n".join(scout_links)
     return (
-        f"Hello,\n\nThe following cases has been uploaded to Scout:\n\n{scout_links_row_separated}"
+        f"Hello,\n\nThe following cases have been uploaded to Scout:\n\n{scout_links_row_separated}"
```
Comment (islean) [inline cg/services/delivery_message/messages/scout_message.py:15]: Good catch!

**Review by islean — APPROVED** [review]
Nice! Should have read it through a bit more carefully when I added the functionality 😓

File: cg/services/delivery_message/messages/statina_message.py [hunk 2]
```diff
@@ -5,5 +5,6 @@
 
 class StatinaMessage(DeliveryMessage):
     def create_message(self, cases: list[Case]) -> str:
-        statina_link: str = get_statina_link(cases[0])
-        return f"Hello,\n\nBatch {statina_link} is now available in Statina."
+        batch_id: str = cases[0].name.split("-")[1]
```
[the remark this answers is not in the document]
Comment (karlnyr) [reply cg/services/delivery_message/messages/statina_message.py:8]: I can do that :)

**Pull request merged**